microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -5,11 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 08/05/2022
+ms.date: 10/26/2022
 ---
 # Create an extension tutorial, part 2
 
-To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  
+To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?tabs=v3).  
```

Comment:
testing the link, which is no longer github.com: it will become:

https://learn.microsoft.com/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?tabs=v3

Reply:
Great, closing